src/github.com/eudat-gef/gef-docker/dckr/dckr.go

```diff
@@ -249,27 +248,65 @@ func (c *Client) BuildImage(dirpath string) (Image, error) {
 // ExecuteImage takes a docker image, creates a container and executes it
 func (c Client) ExecuteImage(id ImageID, binds []string) (ContainerID, error) {
 	img, err := c.c.InspectImage(string(id))
+	//log.Println("img", img)
 	if err != nil {
 		return ContainerID(""), err
 	}
 	hc := docker.HostConfig{
 		Binds: binds,
 	}
+	//log.Println("Config", img.Config);
+	//log.Println("binds", hc.Binds);
 	cco := docker.CreateContainerOptions{
 		Name:       "",
-		Config:     img.Config,
+		Config:     &docker.Config{Image: img.ID,
+						Cmd: []string{"ls", "-l"},
+						OpenStdin:    true,
+						StdinOnce:    true,
+						AttachStdin:  true,
+						AttachStdout: true,
+						AttachStderr: true,
+						Tty:          true,
+					},
+
+
 		HostConfig: &hc,
 	}
-	// fmt.Println("container options", cco)
+	//fmt.Println("container options", cco)
+
 	cont, err := c.c.CreateContainer(cco)
 	if err != nil {
+		log.Println("Create contianer failed")
 		return ContainerID(""), err
 	}
 
+	//defer c.c.StopContainer(cont.ID, 10000)
+
+	//defer
 	err = c.c.StartContainer(cont.ID, &hc)
+	cstatus, err := c.c.WaitContainer(cont.ID)
+
+	fmt.Println("CSTATUS - ", cstatus)
```

Comment:
We should use logs, not prints. If we use prints for debugging, they should not be comitted.